minichlink/minichlink.c

```diff
@@ -2228,44 +2655,23 @@ static int DefaultSetSplit(void * dev, enum RAMSplit split) {
 	return -93;
 }
 
-void PostSetupConfigureInterface( void * dev )
-{
-	struct InternalState * iss = (struct InternalState*)(((struct ProgrammerStructBase*)dev)->internal);
-	iss->nr_registers_for_debug = 32;
-	switch( iss->target_chip_type )
-	{
-	case CHIP_CH32V20x:
-	case CHIP_CH32V30x:
-		iss->sector_size = 256;
-		break;
-	case CHIP_CH32X03x:
-	case CHIP_CH32L10x:
-	case CHIP_CH643:
-		iss->sector_size = 256;  // ??? The X035 datasheet clearly says this is 128 bytes, but fast page erases do 256?
-		break;
-	case CHIP_CH57x:
-	case CHIP_CH56x:
-	case CHIP_CH58x:
-		// Not yet supported here.
-		break;
-	default:
-	case CHIP_CH32V003:
-	case CHIP_CH641:
-		iss->sector_size = 64;
-		iss->nr_registers_for_debug = 16;
-		break;
-	case CHIP_CH32V002:
-	case CHIP_CH32V004:
-	case CHIP_CH32V005:
-	case CHIP_CH32V006:
-		iss->sector_size = 256;
-		iss->nr_registers_for_debug = 16;
-		break;
-	}
-}
-
 int DefaultReadBinaryBlob( void * dev, uint32_t address_to_read_from, uint32_t read_size, uint8_t * blob )
 {
+  fprintf( stderr, "DefaultReadBinaryBlob\n" );
```

Comment:
Tabs not spaces, remove the fprintf stderr, also, ... this logic looks really dubious.